src/test/java/org/drugis/trialverse/dataset/controller/DatasetControllerTest.java

```diff
@@ -186,7 +187,8 @@ public void queryDatasetsRequestPathTurtleType() throws Exception {
 
   @Test
   public void queryDatasetsRequestPathJsonType() throws Exception {
-    List<Dataset> datasets = Arrays.asList(new Dataset("uri", john, "title", "description", "headVersion"));
+    String archivedOn = new Date().toString();
```

Comment:
Test should be expanded to return list of both an archived and non-archived dataset